web/gds-user-ui/src/components/Section/SearchDirectory.tsx

```diff
@@ -93,23 +93,22 @@ const SearchDirectory: React.FC<TSearchDirectory> = ({
               <form onSubmit={(e) => handleSubmit(e, search)}>
                 <FormControl color={'gray.500'}>
                   <HStack>
-                    <AutoComplete
-                      rollNavigation
-                      // no found message
-                      noOptionsMessage="No VASP found">
+                    <AutoComplete rollNavigation>
                       <AutoCompleteInput
-                        variant="filled"
+                        variant="outline"
                         placeholder="Common name or VASP ID"
                         autoFocus
                       />
+
                       <AutoCompleteList>
                         {Object.keys(options)?.map((oid: any, id: any) => (
                           <AutoCompleteItem
                             key={`option-${id}`}
                             value={oid}
                             label={oid}
-                            onClick={() => {
+                            onClick={(e: any) => {
```

Comment:
Would it be possible to clear the result when a user inserts a new search term? In the video below, a result returned for one of my searches as expected. However, when I entered a new search term, I saw the "no results" message but the info for the previous VASP was still listed below the search bar.

https://www.awesomescreenshot.com/video/20205042?key=fb3918a70171fbbbee0db7d3668eb204